### ISSUE TITLE
More info needed with .consist() for detector augment.

### ISSUE THREAD

**Issue by CalocoDoesGit** [cam72cam/ImmersiveRailroading#332]
Me and a friend have been playing around with locomotive automation.
It has come to our attention that the info we require is half given by .info() and half given by .consist().
However things like the Tractive effort is difrent on both commands.

.info() gives you the max tractive effort of the piece of rolling stock that is ontop of the augment, while .consist() gives you the current tractive effort being delliverd by the consist as a whole (all locomotives on the consist combined).
An easy way to seperate this would be to let .consist() or both .info() and .consist() to have Max TE (max the consist/locomotive can give) and Current TE (current TE being produced by the consist/locomotive).

Brake is an .info() only output that only shows the lever position.
It would be nice to have this along with how much force (in Newton) is being put down to hold the consist where it is (or atleast trys) in a simular style to Tractive Effort as mentioned above. (Preferably to both .info() and .consist() )

Same goes for:
- Horespower
- Throttle (lever position)
- Passengers
- Cargo (total item count and % (Posibly table of all the difrent items tho i would either confine that to .info() or a new comand such as .items() for each piece of rolling stock))
- Fluid in mB + % and Max in mb + % (also refer to Cargo)
- Max speed (this is a maybe)

(Quick note: If a detector augment finds nothing it returns nill instead of a table filled with nil values. to circumvent this people have to put all code that takes from the table in an if statement to check if the table actualy exists.... anoyingly.)

**Comment by cloakable** [comment]
Agreed on this; especially for a way to get the cargo on a train. Would be a very useful way to pull a train carrying certain items into a destination interested in those items (iron ingots into a steelworks for example)

**Comment by andrboot** [comment]
If possible to tag-on, if we can include if a wagon/loco is part of a consist the .info the # in the chain?
Would make it easier to handle automation of double-headed locos and such?

**Comment by redstone2019** [comment]
Should this be in "In Progress" not "To Do"?
